### PR TITLE
Update alarm descriptions

### PR DESCRIPTION
**Pull request by BenRamchandani**

### CODE REVIEW

File: terraform/modules/auth_service/monitoring.tf [hunk 1]
```diff
@@ -37,7 +42,12 @@ resource "aws_cloudwatch_metric_alarm" "success_rate_high_login" {
   threshold           = 30 // Highest in last month (10/08) is 23
   treat_missing_data  = "notBreaching"
 
-  alarm_description = "There are more successful logins than expected"
+  alarm_description = <<EOF
+There are more failed logins than expected.
```
Comment (PhoebeJackson1) [inline terraform/modules/auth_service/monitoring.tf:46]: *successful

File: terraform/modules/auth_service/monitoring.tf [hunk 2]
```diff
@@ -117,7 +143,11 @@ resource "aws_cloudwatch_metric_alarm" "auth_forgot_password_rate_limit_reached"
   threshold           = 1
   treat_missing_data  = "notBreaching"
 
-  alarm_description = "An IP address has reached the rate limit on the forgot password page"
+  alarm_description = <<EOF
+An IP address has reached the rate limit on the reset password page
```
Comment (PhoebeJackson1) [inline terraform/modules/auth_service/monitoring.tf:147]: *forgot password

File: terraform/modules/fargate/monitoring.tf [hunk 3]
```diff
@@ -80,7 +80,11 @@ resource "aws_cloudwatch_metric_alarm" "healthy_host_low_alb" {
   statistic           = "Minimum"
   threshold           = var.desired_count
 
-  alarm_description  = "There are fewer healthy hosts than expected"
+  alarm_description  = <<EOF
+There are fewer healthy ECS tasks than expected in the ${aws_ecs_cluster.main.name} cluster.
```
Comment (PhoebeJackson1) [inline terraform/modules/fargate/monitoring.tf:84]: Does healthy hosts = healthy tasks?

Comment (BenRamchandani) [reply terraform/modules/fargate/monitoring.tf:84]: Yeah, I normally say "hosts" to mean EC2 instances and "tasks" to mean ECS tasks, but "host" is more general really.

This alarm is based on the load balancer health checking its targets and it doesn't know what those targets are, HealthyHostCount is a count of how many healthy targets it has.

Comment (BenRamchandani) [reply terraform/modules/fargate/monitoring.tf:84]: Slightly confused by the fact that ECS can do it's own healtchecks that we don't use

**Review by PhoebeJackson1 — APPROVED** [review]

**Pull request merged**